calculator-micro-services/useAlphaFeed.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [ -z "$1" ] 
+then
+  echo "installing default glimpse version"
+  version=""
+elif [ $1 = "--help" ] || [ $1 = "--h" ] || [ $1 = "-h" ] || [ $1 = "-?" ]
+then
+  echo "usage: useAlphaFeed.sh <version>"
+  echo "usage: useAlphaFeed.sh 0.21.4"
+  exit 0
+else
+  echo "installing glimpse version $1"
+  version=@$1
+fi
+
+npm config set @glimpse:registry=https://www.myget.org/F/g-alpha/npm/
```

Comment:
good point, I added logic to save/restore whatever @glimpse:registry is set to.  Note that this is easy enough to do in bash, but not so easy in cmd scripts, so I probably won't do this in PR #6.